test/e2e/logforwarding/syslog/forward_to_syslog_test.go

```diff
@@ -33,90 +35,135 @@ var _ = PDescribe("LogForwarder", func() {
 		e2e              = helpers.NewE2ETestFramework()
 		testDir          string
 		forwarder        *logging.ClusterLogForwarder
+		generatorPayload map[string]string
+		waitlogs         string
+		grepappname      string
+		grepprocid       string
+		grepmsgid        string
+		greptag          string
```

Comment:
nit. golang style is camel case instead of all small caps

Reply:
golang style may be camel case, but these variables are actually bash expressions. these expressions are used with `oc exec` , thats why these are all smallcaps. If these variables were keeping the program state etc, i will use automatically use camel case. but these are different than those.